CMakeLists.txt

```diff
@@ -75,11 +75,11 @@ endif()
 # To insure consistency please ensure the SAME of the first two values are also
 # assigned to the "VERSION" and "BUILD" variables in the native qmake project
 # file, which is NOW called: ./src/mudlet.pro
-set(APP_VERSION 3.22.1)
+set(APP_VERSION 4.0.3)
 if(DEFINED ENV{MUDLET_VERSION_BUILD} AND NOT $ENV{MUDLET_VERSION_BUILD} STREQUAL "")
   set(APP_BUILD $ENV{MUDLET_VERSION_BUILD})
 else()
-  set(APP_BUILD "-dev")
+  set(APP_BUILD "")
```

Comment:
Pretty sure this shouldn't be changes. Same for the pro

Reply:
Could you fix it up? I'm on mobile

Reply:
So am I at the moment. Maybe tomorrow.

Reply:
Fixed.